consensus/src/sync/live/state_queue/mod.rs

```diff
@@ -548,18 +548,23 @@ impl<N: Network> Stream for StateQueue<N> {
         loop {
             match self.chunk_request_component.poll_next_unpin(cx) {
                 Poll::Ready(Some((chunk, start_key, peer_id))) => {
-                    let key = u32::from_str_radix(&format!("{}00000000", start_key)[..8], 16)
-                        .unwrap_or(0);
-
-                    let percentage = (key as f32 / u32::MAX as f32) * 100.0;
-                    log::info!(
-                        ?start_key,
-                        "Received state sync chunk, ~{:.2}% complete",
-                        percentage,
-                    );
-
-                    if let Some(state_chunks) = self.on_chunk_received(chunk, start_key, peer_id) {
-                        return Poll::Ready(Some(state_chunks));
+                    // We no longer need these chunks if we are complete.
```

Comment:
```suggestion
                    // We simply throw remaining incoming chunks away if we are complete.
```
Makes it clearer to me what happens here.